addons/account/models/chart_template.py

```diff
@@ -233,6 +257,13 @@ def _load(self, sale_tax_rate, purchase_tax_rate, company):
         if not self.env.is_admin():
             raise AccessError(_("Only administrators can load a chart of accounts"))
 
+        module_ids = self.env['ir.module.module'].search([
+            ('name', 'in', self._get_extra_modules()),
+            ('state', '=', 'uninstalled')
+        ])
+        if module_ids:
+            module_ids.sudo().button_install()
```

Comment:
@william-andre Not sure why we need this again (this code is there before, so we do it twice?)

Reply:
What do you mean twice?
The post init hook is there **only** to install a localization based on the country of the main company. It is only used for the SaaS, where you define the country before creating the database.
This is there to install all the required modules depending on the CoA being instantiated. It is not doable in the post init hook for reasons explained in the commit message

Reply:
discussed with @oco-odoo, it's ok doing it like that. Another option would have be to define post_init hooks in every loc module that needs it but it's probably better to have all centralized in here